canonical/ubuntu.com
# Update copy and beta sign up form for CUE

## Done

- Updated copy at /credentials
- Created a beta participation form for CUE at /credentials/sign-up

## QA

- Visit the demo at https://ubuntu-com-12237.demos.haus/credentials (should be available whether or not your are logged in)
  - Verify that the copy matches the [copy doc](https://docs.google.com/document/d/1XD9xUamA93hNrVpcUKy70o6UoGOhE15I2Cp8FR5siFI)
- Click on either the "Apply for free beta access!" or "Enroll now >" button to access the beta sign up form
  - Verify that the questions and options match what is in the [spec doc](https://docs.google.com/document/d/1kUwB3coi1BhQ-aCjp4bpqEqamuuXfvgD2M0nKWILd5c)
  - Verify that you can sign up for the beta (Data should be submitted to [Marketo](https://engage-sj.marketo.com/?munchkinId=066-EOV-335#/classic/SL100906930B2LA1) and synced to Salesforce).

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/CRED-88

## Screenshots

![cred-beta-sign-up](https://user-images.githubusercontent.com/995051/200035348-eba942b1-a779-4439-a5a0-75549aa36428.png)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
